src/ImageSharp.Drawing/Processing/LinearGradientBrush.cs

```diff
@@ -137,9 +145,7 @@ protected override float PositionOnGradient(float x, float y)
                     float y4 = y + (k * this.alongX);
 
                     // get distance from (x4,y4) to start
-                    float distance = (float)Math.Sqrt(
-                        Math.Pow(x4 - this.start.X, 2)
-                        + Math.Pow(y4 - this.start.Y, 2));
+                    float distance = (float)Math.Sqrt(Math.Pow(x4 - this.start.X, 2) + Math.Pow(y4 - this.start.Y, 2));
```

Comment:
`MathF`?

Reply:
Hope I won't forget this next time I touch the code.